packages/forklift-console-plugin/src/modules/Providers/views/list/ProvidersListPage.tsx

```diff
@@ -239,6 +244,27 @@ const ModernizeMigration = () => (
   <img src={modernizeMigration} className="forklift-empty-state__icon" />
 );
 
+const handleK8sError = (providersLoaded: boolean, providersLoadError: Error) => {
```

Comment:
why do we need this logic here ?

Reply:
Before this fix, the error messages handling for displayingin page header was missing. 
There were 2 supported scenarios only:
In case of `providersLoadError`, no error msg was was  displayed in providers list page.
In case of of the `inventoryError`, only inventoryNotReachable msg was displayed.

Since this PR fetches both K8s and inventory providers resources and adds more inventory failures error types, I added this handling.
Why do you think it's  unnecessary?